### PR TITLE
Validation: Added Unique Descriptions

### PR DESCRIPTION
**Pull request by JustPies**
Fixed:

- description-missing errors

This PR fixes 30 issues across 30 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-get-print-system-object-properties-without-reflection.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ dev_langs:
 helpviewer_keywords: 
   - "PrintSystemObject [WPF], getting properties"
 ms.assetid: 43560f28-183d-41c1-b9d1-de7c2552273e
+description: Learn how the System.Printing.IndexedProperties namespace provides a means to getting this information without using reflection.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-get-print-system-object-properties-without-reflection.md:10]: It seems some info is missing. It is not clear what "this information" refers to? Are you able to replace "this information" with a noun?

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-make-a-freezable-read-only.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ dev_langs:
 helpviewer_keywords: 
   - "Freezable objects [WPF], making read-only"
 ms.assetid: 6c544b7d-d3c9-4736-aa90-4b8728234ccb
+description: Learn how to make a Freezable read-only by calling its Freeze method, and learn what conditions prevents freezing a Freezable object.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-make-a-freezable-read-only.md:10]: Remove the s from "prevents".

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-override-the-logical-tree.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ helpviewer_keywords:
   - "overriding the logical tree [WPF]"
   - "logical tree [WPF], overriding"
 ms.assetid: 0ae4d074-8113-4b06-b4fa-e0f39d4967a6
+description: Learn how to subclass StackPanel to override the logical tree and enforce a behavior that the panel may only have and will only render a single child element.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-override-the-logical-tree.md:8]: Add a comma based on the correct grouping... I believe after the word "have": 
Learn how to subclass StackPanel to override the logical tree and enforce a behavior that the panel may only have, and will only render a single child element.

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-recognize-application-gestures.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ helpviewer_keywords:
   - "application gestures [WPF], recognizing"
   - "gestures [WPF], recognizing"
 ms.assetid: d58b740f-5192-4a3e-af59-7aa162e6ca15
+description: Learn how to erase ink when a user makes a ScratchOut gesture on an InkCanvas, assuming an InkCanvas, called inkCanvas1, is declared in the XAML file.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-recognize-application-gestures.md:11]: If I am reading this correctly, the comma after the 2nd mention of "InkCanvas" should be removed.

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-remotely-survey-the-status-of-printers.md [hunk 5]
```diff
@@ -11,6 +11,7 @@ helpviewer_keywords:
   - "remotely surveying printer status [WPF]"
   - "status [WPF], printers [WPF], surveying remotely"
 ms.assetid: d6324759-8292-4c23-9584-9c708887dc94
+description: Learn how the printer properties exposed in the APIs of Microsoft .NET Framework provide a means for performing a rapid survey of the states of printers.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-remotely-survey-the-status-of-printers.md:14]: "for performing a rapid survey of the states of printers."... Should states be status?

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-select-ink-from-a-custom-control.md [hunk 6]
```diff
@@ -9,6 +9,7 @@ helpviewer_keywords:
   - "ink [WPF], selecting from custom control"
   - "custom controls [WPF], ink selection"
 ms.assetid: 5f3a45c6-6d40-4017-9b47-933f134ceba3
+descriptions: Learn how adding an IncrementalLassoHitTester to a custom control enables a users to select ink with a lasso tool.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-select-ink-from-a-custom-control.md:12]: "enables a users"...

Remove the "a".

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-override-metadata-for-a-dependency-property.md [hunk 7]
```diff
@@ -9,6 +9,7 @@ helpviewer_keywords:
   - "dependency properties [WPF], overriding metadata for"
   - "overriding metadata for dependency properties [WPF]"
 ms.assetid: f90f026e-60d8-428a-933d-edf0dba4441f
+description: Learn how to override default dependency property metadata from an inherited class by calling the OverrideMetadata method and providing type-specific metadata.
```
Comment (vatmicro) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-override-metadata-for-a-dependency-property.md:12]: For readability I suggest:
Learn how to override "the" default dependency property "of" metadata from an inherited class by calling the OverrideMetadata method and providing type-specific metadata.

**Comment by JustPies** [comment]
**Global effort to fix validation errors**

 @adegeo   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Review by adegeo — APPROVED** [review]

**Pull request merged**